app/mixins/event-wizard.js

```diff
@@ -45,6 +46,10 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
       }
     ];
   },
+  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6140894)